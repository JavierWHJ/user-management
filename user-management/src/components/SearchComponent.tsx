import React from 'react';

const SearchComponent = () => {
    return (
        <div>
            <input 
                // onChange={this.handleSearch} 
                // value={this.state.searchterm} 
                name="searchterm" 
                className="form-control form-control-lg mt-3" 
                placeholder="Search User" 
                type="text" />
        </div>
    );
}

export default SearchComponent;
