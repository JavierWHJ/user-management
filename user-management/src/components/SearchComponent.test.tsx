import React from "react";
import SearchComponent from "./SearchComponent";

import { render } from "@testing-library/react";


const searchUser = (query: string) => {
}

it("matches snapshot", () => {
  const { asFragment } = render(<SearchComponent searchUser={searchUser} />);

  expect(asFragment()).toMatchSnapshot();
});