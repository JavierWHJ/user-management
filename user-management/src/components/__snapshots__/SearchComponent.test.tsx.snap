// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <input
      class="form-control form-control-lg mt-3"
      name="searchterm"
      placeholder="Search User"
      type="text"
    />
  </div>
</DocumentFragment>
`;
